go/worker/storage/committee/checkpoint_sync.go

```diff
@@ -446,10 +445,10 @@ func (n *Node) syncCheckpoints() (*blockSummary, error) {
 				syncState.StateRoot = check.Root
 			}
 
```

Comment:
Update `checkCheckpointUsable` to verify root types. Also check if there are any other places where types should be checked.